monero/wallet_client.go

```diff
@@ -76,6 +76,47 @@ type WalletClientConf struct {
 	LogPath             string               // optional, default is dir(WalletFilePath)/../monero-wallet-rpc.log
 }
 
+// Fill fills in the optional configuration values (Port, MonerodNodes, MoneroWalletRPCPath,
+// and LogPath) if they are not set.
+// If empty, MonerodNodes is set to the first validated node.
```

Comment:
I'd remove the words "If empty". The code is picking the first validated node independent as to whether the nodes were defaulted or not.  I thought it was worth adding to the comments, as it is doing more than just filling.